simulator-client/package.json

```diff
@@ -11,7 +11,7 @@
   "type": "module",
   "dependencies": {
     "net": "^1.0.2",
-    "node-fetch": "3.2.0",
+    "node-fetch": "2.6.7",
```

Comment:
We should be fine with version 2.6.7, according to [GitHub's Advisory Database](https://github.com/advisories/GHSA-r683-j2x4-v87g).

> Affected versions
>= 3.0.0, < 3.1.1
< 2.6.7
Patched versions
3.1.1
**2.6.7**